pylint/testutils/testing_pylintrc

```diff
@@ -7,3 +7,7 @@ disable=
     suppressed-message,
     locally-disabled,
     useless-suppression,
+
+enable=
```

Comment:
We might want to enable them for their functional tests directly. Although it's also a good thing if we have up to date functional tests with only symbolic messages. It's also an internal change for plugin dev but I don't think we advertised that file so we don't need to add a changelog entry (thinking out loud before approving to document that this was considered during review)